pages/home.py

```diff
@@ -0,0 +1,59 @@
+import pandas as pd
+import streamlit as st
+import numpy as np
+import plotly.express as px
+
+@st.cache_data(ttl='1d')
```

Comment:
You'd only want to use this `cache_data` in a scenario where there is data being returned, not the entire main function

Seems specifically the instances where you read `pd.read_csv` -- so we could move those calls into helper functions with the decorator there instead

```
@st.cache_data
def load_lincbrain_datasets():
    return pd.read_csv('lincbrain_datasets.csv')
```    
and

```
@st.cache_data
def load_ lincbrain_modalities():
    return pd.read_csv('lincbrain_modalities.csv')
```

Reply:
Good catch.  I am actually going to remove the caching altogether because we would only want to cache the data until the new data is extracted every night at 2am.  But there is not an easy way to do this natively in Streamlit.  Will reassess in the future if performance is an issue.